streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -296,6 +297,13 @@ public void resume() {
             producer = producerSupplier.get();
             producer.initTransactions();
             recordCollector.init(producer);
+
+            try {
+                stateMgr.checkpoint.delete();
```

Comment:
Is it true that `stateMgr.checkpoint` is always not null here? We should probably condition on that first.

Reply:
yep, right catch.